cmd/bpf2go/main.go

```diff
@@ -70,6 +70,7 @@ func run(stdout io.Writer, pkg, outputDir string, args []string) (err error) {
 
 	fs := flag.NewFlagSet("bpf2go", flag.ContinueOnError)
 	fs.StringVar(&b2g.cc, "cc", "clang", "`binary` used to compile C to BPF")
+	fs.StringVar(&b2g.strip, "strip", "llvm-strip", "`binary` used to strip DWARF from compiled BPF (pass empty string to disable)")
```

Comment:
IMHO the empty string to disable is a bit strange and could be difficult to get right. I would prefer a separate `-no-strip` flag.

Reply:
Good call, I added `no-strip`.